core/executor/Cargo.toml

```diff
@@ -5,27 +5,30 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-derive_more = "0.14.0"
+derive_more = "0.15.0"
 codec = { package = "parity-scale-codec", version = "1.0.0" }
 runtime_io = { package = "sr-io", path = "../sr-io" }
 primitives = { package = "substrate-primitives", path = "../primitives" }
 trie = { package = "substrate-trie", path = "../trie" }
 serializer = { package = "substrate-serializer", path = "../serializer" }
 runtime_version = { package = "sr-version", path = "../sr-version" }
 panic-handler = { package = "substrate-panic-handler", path = "../panic-handler" }
-wasm-interface = { package = "substrate-wasm-interface", path = "../wasm-interface" }
-wasmi = "0.5.0"
+
+# This particular patch version (or higher) is required
+wasmi = "0.5.1"
```

Comment:
+1 for the comment!

Why not `~0.5.1` to actually enforce this though?

Reply:
`^0.5.1` is fine here, I think.